helm/helm-app-database/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../..</relativePath>
+    </parent>
+    <artifactId>helm-app-database</artifactId>
+    <packaging>jar</packaging>
+    <name>Quarkus QE TS: Helm: Application + database</name>
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-openshift</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-container-image-docker</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkiverse.helm</groupId>
+            <artifactId>quarkus-helm</artifactId>
+            <version>0.0.6-SNAPSHOT</version>
```

Comment:
Why did you add `io.quarkiverse.helm:quarkus-helm` to properties `<quarkus-helm.version>0.0.5</quarkus-helm.version>` and than set it here in higher version? And is the snapshot a way to go?

Reply:
[Quarkus Helm is currently using 2.12.2.Final](https://github.com/quarkiverse/quarkus-helm/blob/main/pom.xml#L27), won't it ever cause issues if we use it with f.e. 999-SNAPSHOT?

Reply:
Was a leftover version. The right version was set on the main pom